PhotonVRManager.cs

```diff
@@ -271,13 +226,36 @@ public static void SetCosmetics(Dictionary<string, string> PlayerCosmetics)
         /// Sets a specefic cosmetic
         /// </summary>
         /// <param name="Type">The type of cosmetic you want to set</param>
-        public static void SetCosmetic(string Type, string CosmeticId)
+        public static void SetCosmetic(CosmeticType Type, string CosmeticId)

```

Comment:
this entire function should never have been changed

Reply:
I think that was bc I changed photon vr version 0.0.5 or 0.0.4